lldb/source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2095,6 +2095,50 @@ void SwiftLanguageRuntime::DidFinishExecutingUserExpression(
 
 bool SwiftLanguageRuntime::IsABIStable() { FORWARD(IsABIStable); }
 
+struct AsyncUnwindRegisters {
```

Comment:
Should we add some doxygen comments? Particularly dummy_reg doesn't seem to explain itself :-)

Reply:
maybe it can be named better, what does everyone think about `indirect_marker_regnum`? cc @fredriss

Reply:
(I added doc comments too)